docs/extend/extend-apiml/api-mediation-identity-federation.md

```diff
@@ -0,0 +1,11 @@
+# Zowe API Mediation Layer - Identity Federation
+
+Zowe API ML is able to authenticate users if presented with an access token issued by another (external/distributed) IAM system.
+After the user is authenticated by API ML, the identity of the user encoded with the exteranal access token is federated with the mainframe identity of the user if one exists.
+The mainframe services then receive the mainframe user identity in the API requests as expected according to the configuration of each request.
+
+The Identity federation is based on the SAF Identity Propagation mechanism (external #TODO: Link?).
+The Identity Federation at the API ML level allows for complex deployments made of multiple mainframe systems representing individual security domains, each domain with dedicated API ML installations.
+The mainframe domains can be also independent and detatched.
+
+A prerequisite for the Identity Federation to work is that both external and mainframe user identities be mapped in SAF, and that the distributed identity provider must be trusted by the mainframe security.
```

Comment:
It could link to the api-mediation-oidc-authentication.md document with the ESM instructions, right?

Reply:
Yes.